mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -221,6 +222,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 onGround
 	METHOD method_25936 getLandingBlockState ()Lnet/minecraft/class_2680;
 	METHOD method_27298 shouldSpawnSprintingParticles ()Z
+	METHOD method_29239 dismountVehicle ()V
```

Comment:
We should probably emphasize somehow that you are generally supposed to call `stopRiding` rather than this method

Reply:
Pushed some javadoc, hopefully it's enough.